packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/__snapshots__/ThresholdsFormItem.story.storyshot

```diff
@@ -89,6 +89,22 @@ exports[`Storybook Snapshot tests and console checks Storyshots 2 - Watson IoT E
                 id="downshift-0-toggle-button"
                 onClick={[Function]}
                 onKeyDown={[Function]}
+                title={
+                  <div
+                    className="iot--icon-dropdown__item"
+                    style={
+                      Object {
+                        "color": "#da1e28",
+                      }
+                    }
+                  >
+                    <div
+                      className="iot--icon-dropdown__item-border"
+                    >
+                      <ForwardRef(Warning24) />
+                    </div>
+                  </div>
+                }
```

Comment:
Carbon started using `itemToString` to add a title to the currently selectedItem in a dropdown; however, this breaks our current implementation of IconDropdowns. I've opened an issue with them to add another prop to allow us to render the selectedItem as an element instead of always calling the `itemToString` https://github.com/carbon-design-system/carbon/issues/10038

Reply:
FYI. I think we have had that conversation with them before and they didn't want to support that functionality because they didn't agree with the design use case, but let's hope they change their minds.